Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python:2-slim
-MAINTAINER Avi Aryan <avi.aryan123@gmail.com>
+LABEL maintainer="Avi Aryan <avi.aryan123@gmail.com>"
```

Comment:
Please change this to the fossasia email address or mine.

Reply:
👍